skimage/measure/fit.py

```diff
@@ -642,7 +663,10 @@ def predict_xy(self, t, params=None):
 
         if params is None:
             params = self.params
+
         xc, yc, a, b, theta = params
+        if isinstance(theta, complex):
+            theta = np.real(theta)
```

Comment:
I just saw that `np.real` returns a 0-dimensional array. Do we want to cast to float? And do we want to do this anyway? It might be faster to pipe all values through `float(np.real(theta))` than it is to check first. Thoughts?